README.md

```diff
@@ -1,54 +1,55 @@
 # Soroban CLI (soroban-cli)
 
-This repo is home to the [Soroban CLI](https://github.com/stellar/soroban-tools/tree/main/cmd/soroban-cli): The command-line multi-tool for running and deploying Soroban contracts.
+This repo is home to the Soroban CLI, the command-line multi-tool for running and deploying Soroban contracts on the Stellar network.
 
+## Documentation
 
-## Install the CLI
-Install [the latest soroban-cli version](https://github.com/stellar/soroban-cli/releases).
-`cargo install --locked soroban-cli`
+For installation options see below, for usage instructions [see the manual](/docs/soroban-cli-full-docs.md).
 
-You can also install with `cargo-binstall`.
+## Install
+Install the latest version from source:
 ```
-cargo install cargo-binstall
+cargo install --locked soroban-cli --features opt
```

Comment:
`--features opt` should be the default rec imo for building from source, but folks can always omit it.